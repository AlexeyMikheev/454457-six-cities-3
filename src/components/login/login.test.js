import React from "react";
import renderer from "react-test-renderer";
import Login from "./login.jsx";

it(`Render Login`, () => {
  const tree = renderer
      .create(<Login onSubmit={() => {}} />)
      .toJSON();
  expect(tree).toMatchSnapshot();
});
