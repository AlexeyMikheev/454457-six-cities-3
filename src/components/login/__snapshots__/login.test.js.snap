// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Login 1`] = `
<div
  className="page__login-container container"
>
  <section
    className="login"
  >
    <h1
      className="login__title"
    >
      Sign in
    </h1>
    <form
      action=""
      className="login__form form"
      onSubmit={[Function]}
    >
      <div
        className="login__input-wrapper form__input-wrapper"
      >
        <label
          className="visually-hidden"
          htmlFor="email"
        >
          E-mail
        </label>
        <input
          className="login__input form__input"
          id="email"
          name="email"
          placeholder="Email"
          type="email"
        />
      </div>
      <div
        className="login__input-wrapper form__input-wrapper"
      >
        <label
          className="visually-hidden"
          htmlFor="password"
        >
          Password
        </label>
        <input
          className="login__input form__input"
          id="password"
          name="password"
          placeholder="Password"
          type="password"
        />
      </div>
      <button
        className="login__submit form__submit button"
        type="submit"
      >
        Sign in
      </button>
    </form>
  </section>
  <section
    className="locations locations--login locations--current"
  >
    <div
      className="locations__item"
    >
      <a
        className="locations__item-link"
        xlinkHref="#"
      >
        <span>
          Amsterdam
        </span>
      </a>
    </div>
  </section>
</div>
`;
